modules/subscriptions/views.php

```diff
@@ -150,6 +164,22 @@ static function render_widget_status_messages( $instance ) {
 							__( 'Manage your email preferences.', 'jetpack' )
 						); ?></p>
 					<?php break;
+				case 'many_pending_subs':
+					?>
+					<p class="error">
+						<?php
+						printf(
+							wp_kses(
+								/* translators: 1: Link to Subscription Management page https://subscribe.wordpress.com/, 2: Description of this link */
+								__( 'You already have several pending email subscriptions. <br /> Approve or delete a few subscriptions at <a href="%1$s" title="%2$s" target="_blank" rel="noopener noreferrer">subscribe.wordpress.com</a> before continuing.', 'jetpack' ),
+								self::$allowed_html_tags_for_message
```

Comment:
This is a nice way to keep things tidy and makes it easily reusable in the future. Cool.